src/main/resources/swagger/api-docs.yaml

```diff
@@ -618,6 +620,13 @@ definitions:
         description: |
           The date and time the cluster was last accessed, in ISO-8601 format.
           Date accessed is defined as the last time the cluster was created, modified, or accessed via the proxy.
+      autopause:
```

Comment:
These are in `Cluster` but I think they should be in `ClusterRequest` instead? Currently in Leo `Cluster` is the API _response_ for put/get/list cluster requests.

Reply:
Looks like `ClusterRequest` should have `autopause` and `autopauseThreshold` and `Cluster` should just have `autopauseThreshold` right?